src/server/node_func.c

```diff
@@ -2377,7 +2379,13 @@ decode_Mom_list(struct attribute *patr, char *name, char *rescn, char *val)
 
 	for (i = 0; (p = str_arr[i]) != NULL; i++) {
 		clear_attr(&new, &node_attr_def[(int)ND_ATR_Mom]);
-		if (get_fullhostname(p, buf, (sizeof(buf) - 1)) != 0) {
+		is_node_name_ip = inet_pton(AF_INET, p, &(check_ip.sin_addr)) ;
```

Comment:
Do we need to check for a IPv6 address as well?

Reply:
So, were you going to handle IPv6 as well?

Reply:
In PBS we don't support adding node with their IPv6 address. So i reverted the corresponding changes i made.